src/pkg/services/m365/api/users.go

```diff
@@ -185,26 +185,18 @@ func appendIfErr(errs []error, err error) []error {
 	return append(errs, err)
 }
 
-// EvaluateMailboxError checks whether the provided error can be interpreted
+// IsMailboxErrorIgnorable checks whether the provided error can be interpreted
 // as "user does not have a mailbox", or whether it is some other error.  If
 // the former (no mailbox), returns nil, otherwise returns an error.
```

Comment:
nit: does this comment need to be updated to mention UserNotFound scenario

Reply:
whoops, good catch.